.travis.yml

```diff
@@ -31,6 +31,9 @@ before_install:
 install:
   - travis_retry mvn -B install clean -U -DskipTests=true
 
+after_success:
+  - mvn jacoco:report coveralls:report
```

Comment:
`mvn install` runs `jacoco:report`, I think. If so, `coveralls:report` might be all that is needed.

Even better fix would be to stop running jacoco:report as part of the mainline Maven build.